plasma_cash/root_chain/contracts/DataStructures/Transaction.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.4.18;
+
+import 'ECRecovery.sol';
+import 'RLP.sol';
+
+/**
+* @title Transaction
+*
+* Transaction is used to wrap the Plasma transaction.
+*/
+
+library Transaction {
+    using RLP for bytes;
+    using RLP for bytes[];
+    using RLP for RLP.RLPItem;
+    using RLP for RLP.Iterator;
+
+    struct Tx {
+        uint prevBlock;
+        uint uid;
+        uint amount;
+        address newOwner;
+        address signer;
+    }
+
+    function createTx(bytes memory txBytes, uint numItems)
```

Comment:
I think we can just hardcode 5 inside the function. I don't think there's value for passing the numItems here, it must be 5 anyway, right?

Reply:
Oops. I forgot this. Let me change it.